lightstep-tracer-jre/src/main/java/com/lightstep/tracer/metrics/SafeMetricsImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.lightstep.tracer.metrics;
+
+import com.lightstep.tracer.shared.SafeMetrics;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SafeMetricsImpl implements SafeMetrics {
```

Comment:
Any reason to not use the `SafeMetrics` class in `lightstep-metrics-java` directly?

We also do a few extra checks there, e.g.

```java
    if (accessToken == null) {
      accessToken = "";
    }
```

Reply:
lightstep-metrics-java doesn't have implementation of SafeMetrics interface. Do you want to move SafeMetricsImpl to lightstep-metrics-java?

Reply:
the problem that we need to implement `SafeMetrics` interface which is in java-common. 
but lightstep-metrics-java doesn't depend on java-common.

Reply:
Sure, but maybe you can call it from here? i.e.

```java
public class SafeMetricsImpl implements SafeMetrics {
  @Override
  public Thread createMetricsThread(...) {
    return com.lightstep.tracer.metrics.SafeMetrics.createMetricsThread(...);
  }
}
```

Reply:
I can, but it doesn't use all parameters, e.g. samplePeriodSeconds is ignored and default one is used.

Reply:
default value for serviceVersion and accessToken is  `""`, so we don't need to add check for null.

Reply:
Lets go ahead with this, but let's remove then `SafeMetrics` from `lightstep-metrics-java` (and update the README as well), to avoid having a very similar class in both codebases.

Reply:
yes, I removed SafeMetrics in the PR of lightstep-metrics-java